docs/build/building-apps/04-security-part-1.md

```diff
@@ -0,0 +1,331 @@
+# The Cosmos Security Handbook: Part 1 - Core Chain
+
+> Thank you to **[Roman Akhtariev](https://twitter.com/akhtariev) and [Alpin Yukseloglu](https://twitter.com/0xalpo)** for authoring this post. The original post can be found [here](https://www.faulttolerant.xyz/2024-01-16-cosmos-security-1/).
```

Comment:
While we're at it we could link this https://github.com/crytic/building-secure-contracts/tree/master/not-so-smart-contracts/cosmos

Reply:
good call